Makefile

```diff
@@ -55,10 +62,13 @@ build-mic:clean-mic
 build-demo:clean-demo
 	GOOS=linux GOARCH=amd64 go build -o bin/$(PROJECT_NAME)/$(DEMO_BINARY_NAME) $(GO_BUILD_OPTIONS) github.com/Azure/$(PROJECT_NAME)/cmd/$(DEMO_BINARY_NAME)
 
-build:clean build-nmi build-mic build-demo
+build-identity-validator:clean-identity-validator
+	GOOS=linux GOARCH=amd64 go build -o bin/$(PROJECT_NAME)/$(IDENTITY_VALIDATOR_BINARY_NAME) $(GO_BUILD_OPTIONS) github.com/Azure/$(PROJECT_NAME)/test/e2e/$(IDENTITY_VALIDATOR_BINARY_NAME)
+
+build:clean build-nmi build-mic build-demo build-identity-validator
 
 deepcopy-gen:
-	deepcopy-gen -i ./pkg/apis/aadpodidentity/v1/ -o ../../../ -O aadpodidentity_deepcopy_generated -p aadpodidentity 
+	deepcopy-gen -i ./pkg/apis/aadpodidentity/v1/ -o ../../../ -O aadpodidentity_deepcopy_generated -p aadpodidentity
```

Comment:
Is there a change here ?

Reply:
Yes, my editor removed a trailing whitespace